net-libs/libvncserver/files/libvncserver-0.9.14-crypto-openssl-fix.patch

```diff
@@ -0,0 +1,59 @@
+From b686f379c34114cf938fe88291f58014337558f6 Mon Sep 17 00:00:00 2001
```

Comment:
I'll add a link to the Gentoo bug at the top of the commit & a link to the commit itself.